tests/testapp/migrations/0004_auto_20181129_2100.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 2.1.3 on 2018-11-29 20:00
```

Comment:
rename this file instead, or even better, can you just merged this one with the previous one? You just need to delete them and re-create the migrations, then give it a meaningful name